tests/integration/run.sh

```diff
@@ -17,12 +19,14 @@ echo ''
 echo -e "\033[0;36m#\033[0m"
 echo -e "\033[0;36m# Starting PHP webserver\033[0m"
 echo -e "\033[0;36m#\033[0m"
-PHP_CLI_SERVER_WORKERS=3 php -S localhost:8080 -t ${ROOT_DIR} &
+touch phpserver.log
```

Comment:
To clear on repeating attempts locally:
```suggestion
echo "" > phpserver.log
```
And add to `.gitignore`?

Reply:
Good idea - most of the time I only think about CI...
Should we put the file in `/tmp` to have it outside of the repo?

Reply:
No, inside is good so it's findable 🙈